vita3k/motion/src/motion.cpp

```diff
@@ -19,4 +19,111 @@
 #include <motion/motion.h>
 #include <motion/state.h>
 
-// Put here function used for get motion states
+#include <ctrl/state.h>
+
+#include <SDL_gamecontroller.h>
+
+SceFVector3 get_acceleration(const MotionState &state) {
+    Util::Vec3f accelerometer = state.motion_data.GetAcceleration();
+    return {
+        accelerometer.x,
+        accelerometer.y,
+        accelerometer.z,
+    };
+}
+
+SceFVector3 get_gyroscope(const MotionState &state) {
+    Util::Vec3f gyroscope = state.motion_data.GetGyroscope() * static_cast<float>(2.f * M_PI);
+    return {
+        gyroscope.x,
+        gyroscope.y,
+        gyroscope.z,
+    };
+}
+
+SceFVector3 get_gyro_bias(const MotionState &state) {
+    Util::Vec3f bias = state.motion_data.GetGyroscopeBias();
```

Comment:
You converted gyro input into degrees. Shouldn't this be converted as well since the bias value is in radians?